dataland-frontend/src/components/resources/landing/SampleSection.vue

```diff
@@ -11,7 +11,7 @@
             <p>Preview EU Taxonomy Data of <strong>BMW</strong>, before joining Dataland.</p>
           </div>
           <div class="col-10 text-left p-fluid">
-            <router-link to="/taxonomysample" class="text-primary no-underline font-bold">
+            <router-link to="/samples/eutaxonomy-non-financials" class="text-primary no-underline font-bold">
```

Comment:
Bei der Benennung ist vielleicht zu bedenken, dass zukünftig das Beispiele/ die Beispiele nicht ausschließlich non financial entities sind.

Reply:
Aktuell ist die Komponente selbst tatsächlich auch nur dazu in der Lage, EuTaxonomyNonFinancial Daten anzuzeigen. Ist eine Erweiterung dieser Funktionalität auch im Scope dieses Tickets?

Reply:
Dann lassen wir das so und wenn irgendwann einmal auch andere Companies auf die Samplepage sollen, wird die Route dann gegebenenfalls angepasst.